libraries/ui/src/Card.tsx

```diff
@@ -55,7 +55,7 @@ export const Card: React.FC<CardProps> = ({
 
       <div className="card__content flex flex-col gap-6 w-full flex-1 justify-between">
         <div className="card__text">
-          <h2 className="card__title text-2xl font-[650] text-bluedot-darker mb-1">{title}</h2>
+          <h2 className="card__title text-2xl text-bluedot-darker mb-1">{title}</h2>
```

Comment:
if we update this to the semantically correct h3, we should not need any custom typography styling:
```suggestion
          <h3 className="card__title mb-1">{title}</h3>
```

Reply:
https://github.com/user-attachments/assets/4fcd311e-c9ae-4d1c-82d2-e3f83e68593f